en/config/actuators.md

```diff
@@ -0,0 +1,116 @@
+# Actuator Configuration and Testing
+
+After the airframe is setup you should configure the geometry, assign actuators to outputs, and test the actuator response.
+This can be done in *QGroundControl*, under the **Vehicle Setup > Actuators** tab.
+
+:::note
+This view is only enabled if dynamic control allocation is enabled via [SYS_CTRL_ALLOC](../advanced_config/parameter_reference.md#SYS_CTRL_ALLOC), which will become the default in future.
+If you enable it, also make sure to select the airframe with [CA_AIRFRAME](../advanced_config/parameter_reference.md#CA_AIRFRAME).
+It replaces the existing mixer files and allows configuration via parameters instead.
+:::
+
+This is a screenshot for a multicotor:
+
+![Actuators MC (QGC)](../../assets/config/qgc_actuators_mc_aux.png)
+
+
+:::note
+If you find a setting missing (e.g. reversable motors), make sure to enable the **advanced** checkbox in the top right corner (not visible above).
+:::
+
+## Geometry
+The airframe pre-selects the geometry type, which can provide additional configuration options, such as the motors positions on a multicotor, or the control surface types on a plane.
+The UI reflects that and displays a customized view for each type.
+
+
+### Conventions
+The following sections contain the conventions and explanations for configuring the geometry.
+
+#### Coordinate system
+The coordinate system is NED, where the X axis points forward, the Y axis to the right and the Z axis down.
```

Comment:
NED relative to airframe body or to earth tangent? Might not matter here but if we can be clear, we should be.

Reply:
Updated.